runtime/gc_modron_standard/StandardAccessBarrier.cpp

```diff
@@ -1035,3 +1035,21 @@ MM_StandardAccessBarrier::checkClassLive(J9JavaVM *javaVM, J9Class *classPtr)
 }
 #endif /* defined(J9VM_GC_DYNAMIC_CLASS_UNLOADING) */
 
+bool
+MM_StandardAccessBarrier::preMountContinuation(J9VMThread *vmThread, j9object_t contObject)
+{
+	MM_GCExtensions *extensions = MM_GCExtensions::getExtensions(vmThread->javaVM);
+	if (extensions->isConcurrentScavengerInProgress()) {
+		/* concurrent scavenger in progress */
+		extensions->scavenger->getDelegate()->scanContinuationJavaStack((MM_EnvironmentStandard *)(MM_EnvironmentBase::getEnvironment(vmThread->omrVMThread)), contObject);
+
+	}
+	return true;
+}
+
+bool
```

Comment:
Don't see value in bool ret type, for both pre/post - just make them void.

Don't know why postBatchObjectStore ever had return type bool, but seems like it was never used anywhere or ever returned anything else but true.